src/routes/transfers.js

```diff
@@ -13,9 +13,9 @@ router.put(
 );
 
 router.post(
-  '/update',
+  '/update/:safeAddress',
```

Comment:
Why are you deleting the entire functionality of updating path? @JacqueGM 
I think the functionality might still be useful.

Reply:
mm I thought if we want it, we can add it later?

Reply:
I added it again